dev-tools/cloud/terraform/main.tf

```diff
@@ -58,12 +58,16 @@ resource "ec_deployment" "deployment" {
     "creator" = var.creator
 }
 
-  elasticsearch {}
+  elasticsearch = {
+    hot = {
+      autoscaling = {}
+    }
```

Comment:
Is this empty configuration required? Or are we disabling autoscaling or something? It may be worth of a comment.

Reply:
It comes from the basic example in the terraform provider:
https://registry.terraform.io/providers/elastic/ec/latest/docs/resources/deployment#basic

According to that:
- `hot` is a required parameter of `elasticsearch`
- `autoscaling` is a required parameter of `hot`

So from what I understand, this is the minimum necessary settings for `elasticsearch`.